src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -104,6 +105,10 @@
     <xsd:attribute name="decoration-priority" type="xsd:integer" />
   </xsd:complexType>
 
+  <xsd:complexType name="deprecated">
+    <xsd:attribute name="message" type="xsd:string" />
```

Comment:
rather than using an attribute for the message, why not using the body of the tag ?